openthread-tock/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+
+#  Copyright (c) 2019, The OpenThread Authors.
```

Comment:
This `CMakeLists.txt` is used for building the openthread static library (separate from the app in examples). This is used as the starting point to initiate the openthread build system in conjunction with `create_openthread-tock.sh`.

Since `examples/openthread` directory is separate from this as it links against the static library that is built, I think it makes more sense to keep the openthread library in one place and the application specific implementation in examples.